backend/src/main/java/ddangkong/service/balance/room/RoomService.java

```diff
@@ -0,0 +1,27 @@
+package ddangkong.service.balance.room;
+
+import ddangkong.controller.balance.room.dto.RoomMemberResponse;
+import ddangkong.controller.balance.room.dto.RoomMembersResponse;
+import ddangkong.domain.balance.room.Room;
+import ddangkong.domain.balance.room.RoomRepository;
+import ddangkong.domain.member.MemberRepository;
+import java.util.List;
+import lombok.RequiredArgsConstructor;
+import org.springframework.stereotype.Service;
+
+@Service
+@RequiredArgsConstructor
+public class RoomService {
+
+    private final RoomRepository roomRepository;
+    private final MemberRepository memberRepository;
```

Comment:
의견) 개행을 주는게 컨벤션이니까 통일해주면 좋을 것 같아요~
```suggestion
    private final RoomRepository roomRepository;
    
    private final MemberRepository memberRepository;
```

Reply:
꼼꼼한 컨벤션 체크 감사합니다!